pkg/apis/app/v1/zz_generated.defaults.go

```diff
@@ -1,5 +1,21 @@
 // +build !ignore_autogenerated
 
+/*
+Copyright The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
```

Comment:
@spolti this block should not be getting added when using 0.7.0 sdk. locally when i test against your branch, this block is removed.

Reply:
```shell
$ operator-sdk version
operator-sdk version: v0.7.0
```

Reply:
```shell
$ git diff
diff --git a/pkg/apis/app/v1/zz_generated.defaults.go b/pkg/apis/app/v1/zz_generated.defaults.go
index cce2e60..0b0c84e 100644
--- a/pkg/apis/app/v1/zz_generated.defaults.go
+++ b/pkg/apis/app/v1/zz_generated.defaults.go
@@ -1,21 +1,5 @@
 // +build !ignore_autogenerated
 
-/*
-Copyright The Kubernetes Authors.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
 // Code generated by defaulter-gen. DO NOT EDIT.
 
 package v1
```

Reply:
getting the binary instead building the the sdk works.